testing/coreruleset/coreruleset_test.go

```diff
@@ -131,14 +132,18 @@ func BenchmarkCRSSimplePOST(b *testing.B) {
 }
 
 func TestFTW(t *testing.T) {
-	files := []string{
-		filepath.Join("..", "..", "coraza.conf-recommended"),
-		filepath.Join(crspath, "crs-setup.conf.example"),
-		filepath.Join(crspath, "rules", "*.conf"),
-	}
 	conf := coraza.NewWAFConfig()
-
+	// Configs are loaded with a precise order:
+	// 1. Coraza config
+	// 2. Custom Rules for testing and eventually overrides of the basic Coraza config
+	// 3. CRS basic config
+	// 4. CRS rules (on top of which are applied the previously defined SecDefaultAction)
+	conf = conf.WithDirectivesFromFile(filepath.Join("..", "..", "coraza.conf-recommended"))
 	conf = conf.WithDirectives(`
+SecResponseBodyMimeType text/plain
+SecDefaultAction "phase:3,log,auditlog,pass"
+SecDefaultAction "phase:4,log,auditlog,pass"
```

Comment:
Just like we did in https://github.com/corazawaf/coraza-proxy-wasm/pull/39#discussion_r995572650, I added default actions for phases 3 and 4 because otherwise matched response body rules would still not be logged in the error log, resulting on failing the related tests. I never experienced this necessity against modsec, should there be some default behaviour programmatically set like we have [`const defaultActionsPhase2 = "phase:2,log,auditlog,pass"`](https://github.com/corazawaf/coraza/blob/v3/dev/internal/seclang/rule_parser.go#L23)?

Reply:
Ping @jptosso

Reply:
Shall we take that improvement in a next PR? Maybe add a TODO @M4tteoP and then tackle this research in other PR?

Reply:
+1 For now more passing CRS is better than less so it's probably good to merge and investigate as follow up

Reply:
I think this is a good approach. Let's merge and add a follow-up issue + PR.

Reply:
👍 I'm going to open the issue

Reply:
After increasing this PR，I got the [could not find symbol value for http] error, do you know the reason?